examples/hello_workspaces/Scarb.toml

```diff
@@ -0,0 +1,22 @@
+[workspace]
+members = [
+    "crates/**",
+]
+
+[workspace.scripts]
+test = "protostar --version"
+
+[workspace.dependencies]
+starknet = ">=2.0.0-rc0"
```

Comment:
We're post rc phase in 2.0.0 release line, please revert back to standard version specs